docs/tutorials/gluon/fit_api_tutorial.md

```diff
@@ -0,0 +1,311 @@
+    [Epoch 2] finished in 15.626s: train_accuracy: 0.8375 train_softmaxcrossentropyloss0: 0.4408 <!--notebook-skip-line-->
+    .... <!--notebook-skip-line-->   
+    [Epoch 3] finished in 15.340s: train_accuracy: 0.8575 train_softmaxcrossentropyloss0: 0.3893 <!--notebook-skip-line-->
+    .... <!--notebook-skip-line-->
+    [Epoch 4] finished in 15.420s: train_accuracy: 0.8694 train_softmaxcrossentropyloss0: 0.3560 <!--notebook-skip-line-->
+
+
+## Comparison with Trainer Loop (Older way)
+
+Without the Fit API, the code to train using the Gluon Trainer Loop looks something like this below :
+
+
+```python
+epochs = 5
+
+alexnet_model = get_model('alexnet', pretrained=False, classes = 10, ctx=ctx)
+alexnet_model.initialize(force_reinit=True, init = mx.init.Xavier(), ctx=ctx)
+
+loss_fn = gluon.loss.SoftmaxCrossEntropyLoss()
+learning_rate = 0.04 # You can experiment with your own learning rate here
+
+trainer = gluon.Trainer(alexnet_model.collect_params(), 
+                        'sgd', {'learning_rate': learning_rate})
+
+acc = mx.metric.Accuracy()
+
+# Gluon Training loop goes here 
+
+for epoch in range(epochs):
+    train_acc = 0.0
+    train_loss = 0.0
+    validation_acc = 0.0
+    acc.reset()
+    for data, label in train_iter:
+        data = data.as_in_context(ctx)
+        label = label.as_in_context(ctx)
+        with autograd.record():
+            output = alexnet_model(data)
+            loss_val = loss_fn(output, label)
+            
+        loss_val.backward()
+        trainer.step(batch_size)
+        acc.update(preds = output, labels=label)
+        l = loss_val.mean().asscalar()
+        train_loss += l
+        train_acc += acc.get()[1]
+    
+    print("Epoch %d: , train acc %.3f , train loss %.3f " % (epoch, train_acc/len(train_iter), train_loss/ len(train_iter)))
+
+# Gluon Training loop ends
+```
+
+    Epoch 0: , train acc 0.412 , train loss 1.106 <!--notebook-skip-line-->
+    Epoch 1: , train acc 0.777 , train loss 0.543 <!--notebook-skip-line-->
+    Epoch 2: , train acc 0.832 , train loss 0.439 <!--notebook-skip-line-->
+    Epoch 3: , train acc 0.857 , train loss 0.387 <!--notebook-skip-line-->
+    Epoch 4: , train acc 0.867 , train loss 0.357 <!--notebook-skip-line-->
+
+
+The training loop involves : 
+- Manually iterating over epochs and batches
+- Recording the gradients during the forward pass
+- Computing the loss function
+- Calling the back propagation on the loss function
+- Applying the training step, i.e, updating the weights
+- Recording any useful metrics in the meanwhile
+
+## Summary
+
+In this tutorial, we learnt how to use ```Gluon Fit APIs``` for training a deep learning model and compared it with the existing gluon trainer loop. 
+
+## Next Steps 
+
+- To learn more about deep learning with MXNet Gluon, see [Deep Learning - The Straight Dope](https://gluon.mxnet.io)
+- For more hands on learning about deep learning, checkout out [Dive into Deep Learning](https://d2l.ai)
```

Comment:
Nit: Extra 'out'

Reply:
Done in e44ddaf33e207359217061b2b4c17d723a25f6f0